### ISSUE TITLE
Firefox' read-only bug: duplicate the CSS for it

### ISSUE THREAD

**Issue by binarykitchen** [postcss/autoprefixer#543]
If you look at https://bugzilla.mozilla.org/show_bug.cgi?id=312971 you can see that the latest FF still has issues with the `read-only` selector.

For example I cannot use something like this

```stylus
&:read-only
&:-moz-read-only
    <css code 1>

&:not(:read-only)
&:not(:-moz-read-only)
    <css code 2>
```

as a temporary workaround I have to duplicate the selectors like this:
```stylus
&:read-only
    <css code 1>

&:-moz-read-only
    <css code 1>

&:not(:read-only)
    <css code 2>

&:not(:-moz-read-only)
    <css code 2>
```

would be good if the autoprefixer can do this for me. what do you think?

**Comment by ai** [comment]
I need a W3C spec for `read-only` selector and Can I Use page. Or any resources about browsers support to send PR about `read-only` to Can I Use.

**Comment by binarykitchen** [comment]
Lists supported browsers:
https://developer.mozilla.org/en-US/docs/Web/CSS/%3Aread-only

W3 standard
http://www.w3.org/TR/html5/disabled-elements.html#selector-read-write

**Comment by ai** [comment]
I sent a PR to Can I Use https://github.com/Fyrd/caniuse/pull/2050

**Comment by ai** [comment]
Done https://github.com/postcss/autoprefixer/commit/c447a46f98ef9e19e338d543a467058b893561ab

**Comment by binarykitchen** [comment]
thanks guys

**Comment by FezVrasta** [comment]
When can we expect a new release with this fix inside?

**Comment by ai** [comment]
Today or tomorrow :)

**Comment by FezVrasta** [comment]
Thanks! :)

**Comment by ai** [comment]
Released in 6.1
